package.json

```diff
@@ -30,19 +33,23 @@
     "jdenticon": "^2.2.0",
     "jitsi-iframe-api": "^1.0.0",
     "lodash-es": "^4.17.15",
+    "prerender-spa-plugin": "^3.4.0",
     "promise-polyfill": "8.1.3",
     "soundcloud-widget": "^0.2.1",
     "tipping-contract": "github:aeternity/tipping-contract#develop",
     "vue": "^2.6.12",
     "vue-i18n": "^8.21.1",
     "vue-router": "^3.4.5",
+    "vue-server-renderer": "^2.6.0",
     "vuex": "^3.5.1",
     "vuex-router-sync": "^5.0.0"
   },
   "devDependencies": {
+    "@akryum/vue-cli-plugin-ssr": "~0.6.0",
```

Comment:
Please, investigate the difference between this package and https://www.npmjs.com/package/@uvue/vue-cli-plugin-ssr
https://universal-vue.github.io/docs/guide/#getting-started
https://github.com/universal-vue/uvue

Reply:
Done

Reply:
So, what is the difference between them except that `@akryum/vue-cli-plugin-ssr` is a bit more popular?

Reply:
@davidyuk I messed up with tools because of health. Thanks for notice. I will use https://universal-vue.github.io/docs/guide/#getting-started at the next week and make that ASAP.